packages/stores/src/schema-store/actions/execute-operation.ts

```diff
@@ -19,6 +31,46 @@ import type {
   SchemaStoreActions,
 } from "../schema-store.types";
 
+enum Directive {
+  Defer = "defer",
+}
+
+const usingDefer = (set: SelectionSetNode | undefined): boolean =>
+  set?.selections.some((selection) => {
+    // add kind.FRAGMENT_SPREAD once we support named fragments
+    const inlineFragment = selection.kind == Kind.INLINE_FRAGMENT;
```

Comment:
Any reason not to just include `FRAGMENT_SPREAD` now so we don't have to remember it later?

Reply:
Done

Reply:
Although I'm not certain that the fragment spread will have the selection set inline so this might still need more work, maybe it'd be best to revert to make sure we take a closer look once it's supported

Reply:
Done